.github/workflows/ci.yml

```diff
@@ -39,6 +44,19 @@ jobs:
         with:
           python-version: "2.7.x"
           architecture: x64
+        
+      - name: Install cross compile deps
```

Comment:
No luck with compiling directly on arm?

Reply:
Haven't really tried since I added arm64 builds last time. Last time it was problem of missing build  tool binaries on these platforms. It would also be very slow. Without setting up our own github actions runners the only way to build on arm is qemu emulation.

Reply:
Would it be possible to compile on a Raspberry Pi 4 (I'm mostly concerned about it having only 4GB of RAM).

Reply:
Actually thinking about it -- cross build is probably a lot faster. But it'd be nice to run the tests on real HW.

Reply:
> Would it be possible to compile on a Raspberry Pi 4 (I'm mostly concerned about it having only 4GB of RAM).

It might be I got some versions of v8 to compile fine on a Pi 3, but it's really slow(2-3 hours). Not sure about the Pi 4 though.